app/views/chargeback/_cb_rate_show.html.haml

```diff
@@ -1,4 +1,5 @@
 - breakdown_present = @record.chargeback_rate_details.any? { |d| d.sub_metric.present? }
+= render :partial => 'layouts/flash_msg', :locals => {:flash_div_id => 'exp_editor_flash'} if @flash_array
```

Comment:
That `flash_div_id` and `if @flash_array` look wrong - we never use `if @flash_array` when doing `render :partial`.

Was there a reason this is not simply `render :partial => 'layouts/flash_msg'` ?

Reply:
The reason was to make sure that the message will render properly. But yes, it is not required there, to use  `@flash_array` and `flash_div_id`. We don't need it there. I will change it. Thanks!